public/app/plugins/datasource/cloudwatch/syntax.ts

```diff
@@ -339,7 +339,7 @@ const tokenizer: Grammar = {
     alias: 'function',
   },
   function: {
-    pattern: new RegExp(`\\b(?:${FUNCTIONS.map(f => f.label).join('|')})`, 'i'),
+    pattern: new RegExp(`\\b(?:${FUNCTIONS.map(f => f.label).join('|')})\\b`, 'i'),
```

Comment:
This change looks a bit unrelated.

Reply:
Tangentially related in that it fixes the syntax highlighting for the example queries, but I could put it in a separate PR if you think that'd be better